### PR TITLE
Documented changing display name of group

### PR DESCRIPTION
**Pull request by m7913d**
This instruction was already added to nextcloud in 2019, see commit https://github.com/nextcloud/server/commit/653628c8fb69dc3f9d26751520f91e43a18f17ae

### CODE REVIEW

**Comment by m7913d** [comment]
@nickvergessen Should I change something to get this PR approved?

File: admin_manual/configuration_user/instruction_set_for_groups.rst [hunk 1]
```diff
@@ -166,6 +166,49 @@ XML output
...
+Edit data of a single group
+--------------------------
+
+Edits attributes related to a group. Authentication
+is done by sending a Basic HTTP Authorization header.
+
+**Syntax: ocs/v1.php/cloud/groups/{groupid}**
+
+* HTTP method: PUT
+* PUT argument: key, the field to edit:
```
Comment (nickvergessen) [inline admin_manual/configuration_user/instruction_set_for_groups.rst:178]:
```suggestion
* PUT argument: key, string - the field to edit:
```

File: admin_manual/configuration_user/instruction_set_for_groups.rst [hunk 2]
```diff
@@ -166,6 +166,49 @@ XML output
...
+
+  + displayname
+
+* PUT argument: value, the new value for the field
```
Comment (nickvergessen) [inline admin_manual/configuration_user/instruction_set_for_groups.rst:182]:
```suggestion
* PUT argument: value, string - the new value for the field
```

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
> Should I change something to get this PR approved?

I think people monitoring this repo actively are currently on vacation, so yeah pinging random people was missing :P

**Comment by blizzz** [comment]
@m7913d could you rebase, please?

**Comment by m7913d** [comment]
@blizzz I manually rebased this feature branch to the latest nextloud:master. The /rebase github action seems to fail for some reason.

File: admin_manual/configuration_user/instruction_set_for_groups.rst [hunk 3]
```diff
@@ -166,6 +166,49 @@ XML output
     </data>
   </ocs>  
 
+Edit data of a single group
+--------------------------
```
Comment (nickvergessen) [inline admin_manual/configuration_user/instruction_set_for_groups.rst:170]:
```suggestion
---------------------------
```

**Comment by m7913d** [comment]
@tflidd I fixed the missing DCO. Sorry for the inconvenience.

**Pull request merged**